main.tf

```diff
@@ -117,8 +118,8 @@ resource "aws_elasticache_replication_group" "default" {
 # CloudWatch Resources
 #
 resource "aws_cloudwatch_metric_alarm" "cache_cpu" {
-  count               = var.enabled ? 1 : 0
-  alarm_name          = "${module.label.id}-cpu-utilization"
+  count               = var.enabled ? lenght(local.elasticache_member_clusters) : 0
```

Comment:
```suggestion
  count               = var.enabled ? length(local.elasticache_member_clusters) : 0
```

Reply:
Thanks for the review, updated the code for same,